cmd/telepresence/main.go

```diff
@@ -19,6 +21,8 @@ func main() {
 	}
 
 	cmd := cli.Command(ctx)
+	cmd.AddCommand(daemon.Command())
```

Comment:
This is likely to conflict with what's on release/v2 where I added special handling for the root daemon.